packages/clerk-js/src/ui/components/Checkout/CheckoutForm.tsx

```diff
@@ -182,84 +187,82 @@ const CheckoutFormElements = ({
   return (
     <Col
       elementDescriptor={descriptors.checkoutFormElementsRoot}
-      gap={3}
+      gap={4}
       sx={t => ({ padding: t.space.$4 })}
     >
-      {submitError && (
-        <Alert
-          variant='danger'
-          sx={t => ({
-            animation: `${animations.textInBig} ${t.transitionDuration.$slow}`,
-          })}
+      {/* only show if there are payment sources and there is a total due now */}
+      {paymentSources.length > 0 && checkout.totals.totalDueNow.amount > 0 && (
+        <Box
+        // elementDescriptor={descriptors.checkoutFormPaymentMethodToggle} TODO(@Commerce): add this
         >
```

Comment:
Is this <Box /> needed? It currently does not have any styling associated with it.